apps/android_camera/app/src/main/jni/Application.mk

```diff
@@ -32,9 +32,9 @@ APP_ABI ?= all
 APP_STL := c++_shared
 
 APP_CPPFLAGS += -DTVM_LOG_STACK_TRACE=0 -DTVM4J_ANDROID=1 -std=c++17 -Oz -frtti
-ifeq ($(USE_OPENCL), 1)
-    APP_CPPFLAGS += -DTVM_OPENCL_RUNTIME=1
-endif
+
+# OpenCL support
```

Comment:
I thought that it will be better to enable OpenCL support by default for common applications such as `android_camera` or `android_rpc`. `USE_OPENCL` is specified [here](https://github.com/apache/tvm/pull/13362/files#diff-4f7fde09bab2f87e09bd09c6e4c51c95fc1720dd6917cb239cc579fc68057e82L36-L37). It is a variable which was defined in `android_camera`. I can return this variable and enable build with OpenCL by default.

Reply:
returned verification of USE_OPENCL :+1: